<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>iconsimple.jpg</normaloff>iconsimple.jpg</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stock</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaStock1">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>101</y>
     <width>911</width>
     <height>361</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>Producto</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
